tools/rvn/CommandLineApp.cs

```diff
@@ -373,6 +375,17 @@ private static void ValidateRavenSystemDir(CommandArgument systemDirArg)
             {
                 throw new InvalidOperationException($"Directory does not exist: { systemDirArg.Value }.");
             }
+
+            if (fullPath.EndsWith("System\\"))
```

Comment:
If you will use DirectoryInfo then DirectoryInfo.Name should be what you want IIRC